### ISSUE TITLE
Use native types instead of SDK types

### ISSUE THREAD

**Issue by fmorency** [liftedinit/manifest-app#46]
We're querying the server REST endpoint which returns `SDKType`.

The `SDKType` generated by Telescope breaks the TS typing with numerical enums, e.g., `ProposalStatus`. The query will return the string field equivalent instead of the numerical value, even if the `SDKType` generated by Telescope expects the numerical value.

We should use native types instead.

Sadly, enabling `toSDK` and `fromSDK` in ManifestJS is not enough. Nested `Any` from the `x/group` module is broken.

This is a Telescope issue.

**Comment by fmorency** [comment]
Relates https://github.com/cosmology-tech/telescope/issues/660#issuecomment-2483168929

**Comment by chalabi2** [comment]
is this currently breaking anything in the ui?

**Comment by fmorency** [comment]
No, this is something to be done in the future once Telescope is fixed.
